node/sc/sub_event_handler.go

```diff
@@ -45,6 +45,28 @@ func (cce *ChildChainEventHandler) HandleChainHeadEvent(block *types.Block) erro
 	logger.Trace("bridgeNode block number", "number", block.Number())
 	cce.handler.LocalChainHeadEvent(block)
 
+	// During HandleChainHeadEvent, RPC was sent to inquire the balance of parentOperator,
+	// but RPC send was not processed until chainHeadEvent processing was finished,
+	// and RPC failed due to timeout after event handling was finished.
+	// So, the logic to inquire the parentOperator's balance was handled as a goroutine.
+	go func() {
+		pBalance, err := cce.handler.getParentOperatorBalance()
+		if err != nil {
+			logger.Trace("[SC][View] Failed to get parent operator's balance", "err", err)
+		} else {
+			logger.Trace("[SC][View] Got parent operator's balance", "parentOperator", cce.handler.GetParentOperatorAddr(), "block", block.Number(), "balance", pBalance)
+			parentOperatorBalanceGauge.Update(pBalance.Int64())
+		}
+	}()
+
+	cBalance, err := cce.handler.getChildOperatorBalance()
+	if err != nil {
+		logger.Trace("[SC][View] Failed to get child operator's balance", "err", err)
```

Comment:
```suggestion
		logger.Error("[SC][View] Failed to get child operator's balance", "err", err)
```
How about change the log level up to Error?